danPortfolio.css

```diff
@@ -0,0 +1,349 @@
+@import url('https://fonts.googleapis.com/css2?family=Itim&display=swap');
+
+@font-face {
+    font-family: "Itim";
+    src: url("../fonts/Itim-Regular.ttf") format('truetype');
+}
+
+* {
+	font-family: 'Itim';
+	box-sizing: border-box;
+	
+}
+
+body {
+	align-self: flex-start;
+    margin: 0;
+}
+
+p, h2, h3, h4 {
+	margin: 0;
+}
+
+p, h1, h2, h3, img {
+	margin-bottom: 20px;
+}
+
+html {
+  font-size: 10px;
+}
+
+nav {
+	font-size: 2rem;
+}
```

Comment:
i don't think this styling actually ever applies, because more specific styles for elements e.g. p or a will always precede it